docs/content/posts/2024-07-15-release-1.20.0.md

```diff
@@ -0,0 +1,560 @@
+
+Start from Flink 1.19, the community has decided to officially deprecate multiple APIs that were approaching 
+end of life for a while. In 1.20, we further sorted through all relevant APIs that might need to be replaced
+or deprecated to clear the way for the 2.0 release:
+- Configuration Improvements: As Flink moves towards version 2.0, we have revisited all runtime & Table API/SQL 
+configuration options and identified several opportunities to enhance user-friendliness and maintainability.
+- Deprecate the Legacy [SinkFunction](https://nightlies.apache.org/flink/flink-docs-release-1.20/api/java/org/apache/flink/streaming/api/functions/sink/SinkFunction.html) API: Since its introduction in Flink 1.12, the [Unified Sink API](https://nightlies.apache.org/flink/flink-docs-release-1.20/api/java/org/apache/flink/api/connector/sink2/Sink.html)
+has undergone extensive development and testing. Over multiple release cycles, the API has demonstrated
+stability and robustness, aligning with the criteria set forth in [FLIP-197](https://cwiki.apache.org/confluence/display/FLINK/FLIP-197%3A+API+stability+graduation+process) for API stability graduation.
+Therefore, we promote the Unified Sink API v2 to `@Public` and deprecate the legacy `SinkFunction` interface.
+
+It has been seven years since the Flink community's last major release, and we have great expectations for Flink 2.0.
+We are planning to release several high-impact features in `2.x`. Some of them are already introduced in Flink 1.20 in MVP (minimum viable product) state and discussed in more detail below.
+- Introduce a New Materialized Table for Simplifying Data Pipelines: [FLIP-435](https://cwiki.apache.org/confluence/display/FLINK/FLIP-435%3A+Introduce+a+New+Materialized+Table+for+Simplifying+Data+Pipelines) designed to simplify the development of
+data processing pipelines. With dynamic table with uniform SQL statements and freshness, users can define batch
+and streaming transformations to data in the same way, accelerate ETL pipeline development, and manage task scheduling
+automatically. See below for more details on this exciting feature.
+- Unified File Merging Mechanism for Checkpoints: The unified file merging mechanism for checkpointing is introduced to
+Flink 1.20 as an MVP feature, which allows scattered small checkpoint files to be written into larger files, reducing
+the number of file creations and file deletions and alleviating the pressure of file system metadata management raised by
+the file flooding problem during checkpoints.
+
+# Flink SQL Improvements
+
+## Introduce Materialized Tables
+
+We introduced Materialized Tables abstraction in Flink SQL, a new table type designed to simplify both batch and stream
+data pipelines while providing a consistent development experience.
+
+Materialized tables are defined with a query and a data freshness specification. The engine automatically derives the table
```

Comment:
I was little confused by the phrasing "Flink streaming or batch jobs" . I assume that the Materialized table is a new batch job that can be specified in SQL, it uses the same SQL syntax as stream jobs, but allows you to define when your data is retrieved from you data sources, reducing the number of calls to the data sources. If I have understood this correctly 0 coudl we use these sort of words to be more explicit as to what we are introducing? We could mention how it can be benefieical compared to a view as well.